package.json

```diff
@@ -32,7 +31,7 @@
     "test": "mocha"
   },
   "engines": {
-    "node": ">= 0.10"
+    "node": ">= 10.6"
```

Comment:
could you update it to the LTS at least

Reply:
I updated to >=10.6 as it's LTS at the moment. I've found no 10.6LTS-syntax available...

As the LTS concept is one that's controlled by the Node.js LTS Working Group, which is a separate team from the npm CLI, this is likely to be tricky to get into npm. The CLI team isn't going to have time to get to this in the near-to-medium term.

Reply:
LTS for version 6 or 8 should be more flexible for users
https://github.com/nodejs/Release#release-schedule